--- a/lib/perl5/Catalyst/Test.pm	2011-02-16 22:28:46.000000000 +0100
+++ b/lib/perl5/Catalyst/Test.pm	2012-02-09 23:09:30.000000000 +0100
@@ -305,17 +305,17 @@
             # getting it into Plack::Test, or make a middleware out of it, or
             # whatever. Seriously - horrible.
 
-            require HTML::HeadParser;
+            #require HTML::HeadParser;
 
-            my $parser = HTML::HeadParser->new();
-            $parser->xml_mode(1) if $resp->content_is_xhtml;
-            $parser->utf8_mode(1) if $] >= 5.008 && $HTML::Parser::VERSION >= 3.40;
-
-            $parser->parse( $resp->content );
-            my $h = $parser->header;
-            for my $f ( $h->header_field_names ) {
-                $resp->init_header( $f, [ $h->header($f) ] );
-            }
+            #my $parser = HTML::HeadParser->new();
+            #$parser->xml_mode(1) if $resp->content_is_xhtml;
+            #$parser->utf8_mode(1) if $] >= 5.008 && $HTML::Parser::VERSION >= 3.40;
+
+            #$parser->parse( $resp->content );
+            #my $h = $parser->header;
+            #for my $f ( $h->header_field_names ) {
+            #    $resp->init_header( $f, [ $h->header($f) ] );
+            #}
             # Another horrible hack to make the response headers have a
             # 'status' field. This is for back-compat, but you should
             # call $resp->code instead!
